### PR TITLE
Updating OWNERS file

### PR DESCRIPTION
**Pull request by Edwinhr716**
#### What type of PR is this?
/kind cleanup

#### Which issue(s) this PR fixes

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -3,9 +3,10 @@
 approvers:
   - ahg-g
   - kerthcet
-  - liurupeng
 
 reviewers:
   - ahg-g
   - kerthcet
```
Comment (ahg-g) [inline OWNERS:9]: can you add yourself as a reviewer please?

Comment (Edwinhr716) [reply OWNERS:9]: added

**Pull request merged**